mta-sts.md

```diff
@@ -498,4 +498,34 @@ https://mta-sts.example.com/.well-known/mta-sts.json:
 
 ~~~~~~~~~
 
+# Appendix 2: Sending MTA State Machine
+
+It is helpful to think of the different states of a sending MTA and a receiving
```

Comment:
Seems correct to me.

To be clear, with case 2, I think it implies we don't replace policies,
which is not exactly true. The subtlety here is that we *replace* policies
only once the *new policy* is successfully applied. (At least, that's the
implication of requiring policies to be successfully applied before
enforcement and how replacement is described, I think. Maybe this is not so
clear in the spec, though...hmmm...)

On Tue, Sep 13, 2016 at 6:13 PM jjones425 <notifications@github.com> wrote:

> In mta-sts.md
> <https://github.com/mrisher/smtp-sts/pull/103#discussion_r78592266>:
>
> > @@ -498,4 +498,34 @@ https://mta-sts.example.com/.well-known/mta-sts.json:
> >
> >  ~~~~~~~~~
> >
> > +# Appendix 2: Sending MTA State Machine
> > +
> > +It is helpful to think of the different states of a sending MTA and a receiving
>
> This works. It might be useful to have a decision tree diagram, but, not
> sure if diagrams are something commonly used.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mrisher/smtp-sts/pull/103/files/ea1fb592e4efc3ad5d1a840d6e59ea510d953f46#r78592266>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1vi5xZE4O55HJxZD2BoyrIOiYnnmGVks5qpsuLgaJpZM4J6C5r>
> .
>